insights-client.spec

```
@@ -43,6 +43,16 @@ BuildRequires: systemd-rpm-macros
 %description
 Sends insightful information to Red Hat for automated analysis
 
+%package ros
+Requires: pcp-zeroconf
+Summary: The subpackage for Resource Optimization Service
```

This should probably declare are relation to Red Hat Insights somehow. Maybe:
```suggestion
Summary: Insights Resource Optimization Service
```

---

Branding rules mandate only "Insights" should be capitalized:

```suggestion
Summary: The subpackage for Insights resource optimization service
```